README.md

```diff
@@ -103,6 +103,12 @@ There are two types of configuration that `wrangler` uses: global user and per p
     $ brew install openssl@1.1
     ```
 
+1. Update
+    To get the latest version of Wrangler run:
+  
+    ```bash
+    cargo install --git https://github.com/cloudflare/wrangler --force
+    ```
```

Comment:
I'm not sure but `npm install @cloudflare/wrangler` should do the trick too because it's taking the latest release on GitHub

Reply:
Are there different workflows for someone using cargo ? Or is everyone supposed to use npm now?

Reply:
We should keep both workflows (cargo or npm).